File: pset6/mario/mario.py
from cs50 import get_int

# Promt for pyramin height between 1 and 23
nPHeight = -1
while nPHeight < 0 or nPHeight > 23:
    nPHeight = get_int("Enter positive number no greater than 23: ")

# For each line
for i in range(nPHeight):
    # White space and blocks
    print(" " * - (i + 1 - nPHeight), "#" * (i + 2), sep='', end='')
    print('')

File: pset1/mario/less/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)

{
    int nPHeight;

    //Promt for pyramin height
    do
    {
        nPHeight = get_int("Enter positive number no greater than 23: ");
    }
    while ( nPHeight < 0 || nPHeight > 23 );

    for (int i = 0; i < nPHeight; i++)
    {
        //For each line draw Pyramin height minus line number minus 1 spaces
        for (int j = nPHeight - i - 1; j > 0; j--)
        {
            printf(" ");
        }
        //For each line draw line number plus 2 hash
        for (int k = 0; k < i + 2; k++)
        {
            printf("#");
        }
        printf("\n");
    }

}